### DISCUSSION TITLE
not applied style pagination tailwind

### DISCUSSION THREAD

**Discussion by Kenny201** [gilbitron/laravel-vue-pagination#186] [Help & support]
Please tell me how to enable pagination but styles are not applied
<img width="315" alt="image" src="https://github.com/gilbitron/laravel-vue-pagination/assets/24923249/7fa3ad67-957e-431a-8509-d8ccb32f2e32">
<img width="1136" alt="image" src="https://github.com/gilbitron/laravel-vue-pagination/assets/24923249/75291ce7-6857-44c2-a42e-3226926550d1">

**Comment by DimulyaMonster** [comment]
Same story - if anyone uses this - please assist.
